### ISSUE TITLE
[Driftory website] Sign in requires refreshing the page manually

### ISSUE THREAD

**Issue by Dan503** [pixfabrik/driftory-viewer#36]
After going through the Twitter sign in process, you need to manually refresh the page in order to actually sign in.

If a page refresh is required, can you automatically do a `location.reload()` after you recieve the auth token back from Twitter?

That would make the sign in process less confusing.

(A continuation of [issue 70 in the BIC project](https://daniel-tonon.atlassian.net/browse/BIC-70))

**Comment by iangilman** [comment]
It looks like this might have to do with HTTPS… If I sign in from https://www.driftory.com/create/, I encounter this issue, whereas if I sign in from http://www.driftory.com/create/ (not HTTPS), I do not. I see that regardless of where I sign in from, the twitter callback URL is not HTTPS. I'll look into fixing this based on that.

**Comment by Dan503** [comment]
> the twitter callback URL is not HTTPS

Wait, does that mean you have been getting people to sign into their twitter accounts over a non https connection? 😬😨

**Comment by iangilman** [comment]
No… They sign in on Twitter's site using HTTPS, so their login info is secure, but when they come back to Driftory, it's been via HTTP.

**Comment by iangilman** [comment]
Well, now I've made it so it works properly with HTTPS, but you have to refresh the page if you do it via HTTP. Really, there's no reason for us to continue supporting HTTP, though... the fix should be to auto-redirect anyone coming in via HTTP to HTTPS.
